gen/src/MPIgenerator.jl

```diff
@@ -19,11 +19,7 @@ module MPIgenerator
 
         options = load_options(joinpath(@__DIR__, "generator.toml"))  # wrapper generator options
         options["general"]["callback_documentation"] = node -> String[
-            """
-            `$(node.id)` man page:
-            - [OpenMPI](https://www.open-mpi.org/doc/current/man3/$(node.id).3.php)
-            - [MPICH](https://www.mpich.org/static/docs/latest/www3/$(node.id).html)
-            """
+            "- man page: [OpenMPI](https://www.open-mpi.org/doc/current/man3/$(node.id).3.php), [MPICH](https://www.mpich.org/static/docs/latest/www3/$(node.id).html)"
```

Comment:
Why not using `_doc_external` as [already suggested](https://github.com/JuliaParallel/MPI.jl/pull/652#discussion_r984781481)?

Reply:
I'm not sure what you. You just need to print the string `_doc_external`.

Reply:
Nice, but I can't get rid of the backslash in the docstring ... any hint ?

Reply:
Fixed.